stories/helper_components/Menu.tsx

```diff
@@ -19,6 +18,8 @@ import {
     COLUMN_TOTAL_1ST_2ND_ATTR_MAX,
     GRAND_TOTAL_SUM,
 } from "../data/componentProps";
+import { AFM } from "@gooddata/typings";
+import noop = require("lodash/noop");
```

Comment:
Please move 3rd party library imports to the top.

Reply:
Fixed.